service/src/main/java/tennisclub/service/RankingServiceImpl.java

```diff
@@ -0,0 +1,49 @@
+package tennisclub.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import tennisclub.dao.RankingDao;
+import tennisclub.entity.Tournament;
+import tennisclub.entity.User;
+import tennisclub.entity.ranking.Ranking;
+
+import java.util.List;
+
+@Service
+public class RankingServiceImpl implements RankingService {
+
+    private final RankingDao rankingDao;
+
+    @Autowired
+    RankingServiceImpl(RankingDao rankingDao) { this.rankingDao = rankingDao; }
```

Comment:
I think this constructor should be `public`, not `package` private 🤔

Reply:
Good spot, thanks! That's the result of not writing the tests myself and a lack of perfectionism 😄